composer.json

```diff
@@ -153,8 +153,7 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
         ],
         "create-test-db": [
-            "@php ./tests-legacy/create-test-db.php",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache"
```

Comment:
Why did you remove it ?

Reply:
https://github.com/PrestaShop/PrestaShop/pull/14894/files#r310165191